spec/requests/image_searches_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
 describe '/search/images' do
-  fixtures :affiliates, :instagram_profiles
+  fixtures :affiliates, :flickr_profiles
```

Comment:
I decided to swap flickr in here to preserve the images use case. Open to other ideas of this isn't appropriate.

Reply:
👍